src/main/java/com/twilio/sdk/resource/InstanceResource.java

```diff
@@ -153,6 +153,16 @@ public Integer getPropertyAsInteger(String name) {
 	}
 
 	/**
+	 * Gets the property as a Boolean value.
+	 *
+	 * @param name property name
+	 * @return Boolean value
+	 */
+	public Boolean getPropertyAsBoolean(String name) {
+		return Boolean.valueOf(getProperty(name));
```

Comment:
Then your are completely missing my point. `getProperty` will throw an exception when the json value is a boolean. That means `getPropertyAsBoolean` will also throw an exception because it relies on `getProperty`